source/tutorial/create-system-dsn.txt

```diff
@@ -12,41 +12,161 @@ Create a System DSN
+         - :manual:`Create a MongoDB database </core/databases-and-collections>` to which you want to connect 
+           using |bi-short|.
+
+     - id: ldap
+       name: LDAP
        content: |
 
-Prerequisites
--------------
+         - Make sure your :binary:`~bin.mongod` and 
+           :binary:`~bin.mongos` instances are running 
+           `MongoDB Enterprise <http://www.mongodb.com/products/mongodb-enterprise?jmp=docs>`__. 
+         - Configure your :binary:`~bin.mongod` and 
+           :binary:`~bin.mongos` instances to use 
+           :manual:`LDAP (PLAIN) authentication </core/security-ldap>`.
+         - :manual:`Create a MongoDB database </core/databases-and-collections>` to which you want to connect 
+           using |bi-short|.
+
+     - id: kerberos
+       name: Kerberos
+       content: |
 
-Before creating a :abbr:`DSN (Data Source Name)`, you should:
+         - Make sure your :binary:`~bin.mongod` and 
+           :binary:`~bin.mongos` instances are running 
+           `MongoDB Enterprise <http://www.mongodb.com/products/mongodb-enterprise?jmp=docs>`__. 
+         - Configure your :binary:`~bin.mongod` and 
+           :binary:`~bin.mongos` instances to use 
+           :manual:`Kerberos (GSSAPI) authentication </core/kerberos>`.
+         - :manual:`Create a MongoDB database </core/databases-and-collections>` to which you want to connect 
+           using |bi-short|.
+         - `Create a keytab file`_ on the |kdc| that has the needed 
+           Kerberos :ref:`service principals <kerberos-service-principal>` 
+           for ``mongosql`` and ``mongosql2``.
+
+           .. example::
+
+              .. code-block:: sh
+              
+                 mongosql/<fullHostnameOfBIConnector>@<KERBEROS_REALM>
+ 
+         - Copy the ``keytab`` file you created for your MongoDB 
+           deployment to the same host that serves the |bi-short|. 
+         - If your keytab does not use the default name 
+           (``krb5.keytab``), you must set the 
+           `KRB5_KTNAME <https://web.mit.edu/kerberos/krb5-latest/doc/basic/keytab_def.html>`_ system variable.
+
+           .. note::
+
+              The Kerberos implementation on macOS, `Heimdal`_, does
+              not support the ``default_keytab_name`` configuration 
+              setting, like MIT Kerberos does. You must set the 
+              ``KRB5_KTNAME`` system variable. 
+
+         - If your Kerberos configuration file does not use the 
+           default name (``krb5.conf``), you must set the
+           `KRB5_CONFIG <https://web.mit.edu/kerberos/krb5-devel/doc/admin/conf_files/krb5_conf.html>`_ 
+           system variable. (Heimdal also supports 
+           `this system variable <https://www.h5l.org/manual/HEAD/info/heimdal/Configuration-file.html#Configuration-file>`_.)
+
+         - If you are using a ``mongosqld`` configuration file, you
+           must add the following settings.
+
+           **MongoDB BI Connector Authentication**
```

Comment:
I think this section doesn't look that good.  I suggest the following:

* Use a `.. seealso::` for just the configuration file settings, not their values.  I think the example does a good enough job covering this.
* Ditch the `.. example:` wrapper and just put the code block immediately after "...you must add the following settings:"